### ISSUE TITLE
Increasing Accuracy 

### ISSUE THREAD

**Issue by adithyakumarcr** [sagarVL/TUHH_ISM_tumor_detection#19]
Currently SVM and Linear regression is in the classifier.py program, under classifier_adithya branch. It needs to be fine tuned. Currently we are having an accuracy of 59%(Linear Regression) and 60% (SVM) which is pretty bad. We can expect it to go till 70-75% with Logistic regression and more than 83% with SVM. The values/features are Standardlize the data.

_Before running the program, make sure the data is accessible. Make sure they are in the right directory before launching the program._

There are different parameter that needs to be fine tuned with, such as:

**Logistic Regression:** 
- max_iterations (max_iter)
- solver 
- multi_class
- class_weight
- random_state
- penalty
- C

**SVM**
- probability 
- gamma
- tol
- cache_size
- class_weight 
- random_state
- decision_function_shape
- kernel
- C

https://github.com/data-metrics/skin-lesion-classification/blob/master/models/stacking_model.py 

This page can help you with understanding different types of options that can be fiddled around with. 

Also, A detailed description of the above are avaiable on Sckit websites. (links below)

**Logistic Regression**
https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LogisticRegression.html

**SVM**
https://scikit-learn.org/stable/modules/generated/sklearn.svm.SVC.html
